packages/lib-utils/src/types/k8s.ts

```diff
@@ -1,4 +1,4 @@
-type K8sResourceIdentifier = {
+export type K8sResourceIdentifier = {
```

Comment:
I'm using this in the K8s `~` reference logic... Ideally we wouldn't expose this

Reply:
Hmmm... yeah, we may need tech debt story to clean this up